src/lib/addons/slack-app.test.ts

```diff
@@ -26,7 +26,7 @@ jest.mock('@slack/web-api', () => ({
 }));
 
 describe('SlackAppAddon', () => {
-    let addon;
+    let addon: SlackAppAddon;
```

Comment:
Shouldn't the type be `| undefined`? because you're not assigning it a value on declaration